src/sentry/static/sentry/app/components/stackedBarChart.jsx

```diff
@@ -412,8 +412,6 @@ const StyledSvg = styled('svg')`
 const StyledFigure = styled('figure')`
   display: block;
   position: relative;
-  width: 100%;
-  height: 100%;
```

Comment:
Aha.

Reply:
yeah—when i was writing it, this seemed harmless enough, because `width` and `height` have been set inline on this for a while via props. But it turns out sometimes we set the width and height in our stylesheets.